calibration/experiments/gcm_driven_scm/Manifest.toml

```diff
@@ -1,17 +1,18 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.10.5"
+julia_version = "1.11.5"
```

Comment:
I didn't even know these manifests existed - I assume they didn't need to use Julia 1.10 but they're just old, so this is probably fine